kernel/src/fs/utils/inode.rs

```diff
@@ -434,6 +495,45 @@ pub trait Inode: Any + Sync + Send {
     fn extension(&self) -> Option<&Extension> {
         None
     }
+
+    ///  Used to check for read/write/execute permissions on a file.
+    ///
+    ///  We use "fsuid" for this, letting us set arbitrary permissions
+    ///  for filesystem access without changing the "normal" uids which
+    ///  are used for other things..
```

Comment:
I found this API documentation at https://elixir.bootlin.com/linux/v6.0.9/source/fs/namei.c#L381 with no modification, which violates our license. Can we revise it to be more descriptive and accurate?

Reply:
Done